soroban-env-host/src/native_contract/token/event.rs

```diff
@@ -21,12 +23,14 @@ pub(crate) fn decr_allow(
     e: &Host,
     from: Address,
     to: Address,
+    name: Bytes,
```

Comment:
Nit: I would probably just directly use `read_name` here as it's independent of the event (also would be easier to cleanup in case we decide to use a different approach). But that's up to your judgement.

Reply:
Good point. I was thinking it'd be valuable to be able to see exactly what is emitted from the contract, but there's not much value in that here. Updated.